libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/QnAMaker.java

```diff
@@ -3,7 +3,6 @@
 
 package com.microsoft.bot.ai.qna;
 
-import com.microsoft.bot.ai.qna.QnAMakerOptions;
 import com.microsoft.bot.ai.qna.models.FeedbackRecords;
 import com.microsoft.bot.ai.qna.models.QueryResult;
 import com.microsoft.bot.ai.qna.models.QueryResults;
```

Comment:
I cannot point you the line, between the imports, there is one with the import java.util.*; (line 24) we should add individual imports